themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -644,6 +644,8 @@ client-auth.client_secret_jwt=Client secret as jwt
 client-auth.private_key_jwt=JWT signed with private key
 identity-provider.client-id.tooltip=The client or client identifier registered within the identity provider.
 client-secret=Client Secret
+client-assertion-signing-algorithm=Client Assertion Signature Algorithm
+client-assertion-signing-algorithm.tooltip=JWA Algorithm used for client assertions.
```

Comment:
Is it possible to add slightly more details to this tooltip to specify what exactly does this option mean, when it is needed to be configured etc? My vote is just include everything, which you added to the documentation to this tooltip. So something like this: `Signature algorithm to create JWT assertion as client authentication. In the case of JWT signed with private key or Client secret as jwt, it is required. If no algorithm is specified, the following algorithm is adapted. `RS256` is adapted the in the case of JWT signed with private key.  `HS256` is adapted the in the case of Client secret as jwt.`

Reply:
That's exactly what you say. 
I added the same content as the documentation to tooltip message.